src/legacy/core_plugins/kibana/public/management/sections/objects/saved_object_view.tsx

```diff
@@ -0,0 +1,171 @@
+import { ISavedObjectsManagementRegistry } from '../../saved_object_registry';
+import { Header, NotFoundErrors, Intro, Form } from './components/object_view';
+import { canViewInApp } from './lib/in_app_url';
+import { SubmittedFormData } from './types';
+
+interface SavedObjectEditionProps {
+  id: string;
+  serviceName: string;
+  serviceRegistry: ISavedObjectsManagementRegistry;
+  capabilities: Capabilities;
+  overlays: OverlayStart;
+  notifications: NotificationsStart;
+  notFoundType?: string;
+  savedObjectsClient: SavedObjectsClientContract;
+}
+
+interface SavedObjectEditionState {
+  type: string;
+  object?: SimpleSavedObject<any>;
+}
+
+export class SavedObjectEdition extends Component<
+  SavedObjectEditionProps,
+  SavedObjectEditionState
+> {
+  constructor(props: SavedObjectEditionProps) {
+    super(props);
+
+    const { serviceRegistry, serviceName } = props;
+    const type = serviceRegistry.get(serviceName)!.service.type;
+
+    this.state = {
+      object: undefined,
+      type,
+    };
+  }
+
+  componentDidMount() {
+    const { id, savedObjectsClient } = this.props;
+    const { type } = this.state;
+    savedObjectsClient.get(type, id).then(object => {
+      this.setState({
+        object,
+      });
+    });
+  }
+
+  render() {
+    const {
+      capabilities,
+      notFoundType,
+      serviceRegistry,
+      id,
+      serviceName,
+      savedObjectsClient,
+    } = this.props;
+    const { type } = this.state;
+    const { object } = this.state;
+    const { edit: canEdit, delete: canDelete } = capabilities.savedObjectsManagement as Record<
+      string,
+      boolean
+    >;
+    const canView = canViewInApp(capabilities, type);
+    const service = serviceRegistry.get(serviceName)!.service;
+
+    return (
+      <div
+        className="kuiViewContent kuiViewContent--constrainedWidth"
+        data-test-subj="savedObjectsEdit"
+      >
+        <Header
+          canEdit={canEdit}
+          canDelete={canDelete}
+          canViewInApp={canView}
+          type={type}
+          onDeleteClick={() => this.delete()}
+          viewUrl={service.urlFor(id)}
+        />
+        {notFoundType && (
+          <>
+            <EuiSpacer size="s" />
+            <NotFoundErrors type={notFoundType} />
+          </>
+        )}
+        {canEdit && (
+          <>
+            <EuiSpacer size="s" />
+            <Intro />
+          </>
+        )}
+        {object && (
+          <>
+            <EuiSpacer size="m" />
+            <Form
+              object={object}
+              savedObjectsClient={savedObjectsClient}
+              service={service}
+              editionEnabled={canEdit}
+              onSave={this.saveChanges}
+            />
+          </>
+        )}
+      </div>
+    );
+  }
+
+  async delete() {
+    const { id, savedObjectsClient, overlays, notifications } = this.props;
+    const { type, object } = this.state;
+
+    const confirmed = await overlays.openConfirm(
```

Comment:
This confirm modal for deletion really needs to show the danger colored button which is jut a prop on the EuiModal `buttonColor="danger"`. 

Also, is it at all possible to change the title of this modal to display the name of the object?
<img width="547" alt="Screen Shot 2020-03-09 at 14 25 54 PM" src="https://user-images.githubusercontent.com/549577/76246318-d6734280-6213-11ea-9228-2400749df0e8.png">

@gchaps Perhaps you can also help with the body text of that modal?

Reply:
Delete [Flights] Overview?

This action permanently removes the object from Kibana.

Reply:
We did not have the `buttonColor` option in our core `overlay.openConfirm` API. I added it.

<img width="449" alt="Screenshot 2020-03-10 at 08 41 46" src="https://user-images.githubusercontent.com/1532934/76290207-0c173a80-62ab-11ea-8fa4-70b291e5a967.png">

Should we wrap the title in `"..."` or `'...'` ?

Reply:
I'll leave the quoting question to @gchaps to answer. But can you also update the the body of the modal to:
`This action permanently removes the object from Kibana.`

Reply:
You can wrap the object title in single quotes '...'

Reply:
Done

<img width="498" alt="Screenshot 2020-03-11 at 11 19 42" src="https://user-images.githubusercontent.com/1532934/76406605-4b688880-638a-11ea-9632-189d4ca39978.png">